### DISCUSSION TITLE
benchmarks/

### DISCUSSION THREAD

**Discussion by giscus** [zmike/zmike.github.io#12] [Announcements]
# benchmarks/

I read the article

https://www.supergoodcode.com/benchmarks/

**Comment by rhysperry111** [comment]
You can't just let radeonsi beat you like that... right? If you're not at least 110% better in every benchmark imaginable zink is just useless /s

**Comment by NHOrus** [comment]
[The article](https://www.phoronix.com/review/radeon-zink-summer23), for those who don't know WTF.

**Comment by n8pjl** [comment]
I'd say that you're doing great, given that zink supports much more than just AMD hardware.

**Comment by orowith2os** [comment]
*gets popcorn*

**Comment by qwertychouskie** [comment]
Which one?  The one where AMDVLK beat RADV, or the one where RadeonSI beat Zink+RADV?

**Comment by Dantali0n** [comment]
"But no doubt this positioning is likely to improve once Blumenkrantz reads this article" made me laugh :)

**Comment by EliasOfWaffle** [comment]
But its just a perfomance issues with RADV driver yet?
In your last presentation in Xorg Conference the NVK driver are the are most faster in Zink.
If benchmark will be in ANV,AMDVLK, AMDGPU-PRO or Nvidia hat performance can be better? 
But for now it's have a great evolution, in my test in the past year with Nvidia Proprietary Driver, that performance is good and can be used for daily use

**Comment by ChaosFenix** [comment]
So next update, or maybe several updates, is probably going to be attacking performance in Bioshock Infinite, ET Legacy, and GL Mark 2.

**Comment by ChaosFenix** [comment]
@Mike Blumenkrantz. Side note here. I don't know if this is a setting you have control over but the chat seems to be automatically upvoting myself. I try unchecking it and a couple seconds later it rechecks it. I don't know if this is also happening with the other commenters but given everyone has an upvote it may be the case. Please don't spend too much time on this. I prefer cool Zink articles to fixing a minor bug that has no actual impact on the real world.

**Reply by zmike** [reply]
I think that's just part of how github discussions work
